clustering/status.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/cybozu-go/moco/pkg/constants"
 	"github.com/cybozu-go/moco/pkg/dbop"
 	"github.com/cybozu-go/moco/pkg/password"
+	"golang.org/x/exp/slices"
```

Comment:
Can't we use the standard package?
https://pkg.go.dev/slices

Reply:
Thank you! I don't know the standard package. I'll use it.

Reply:
fixed 3b9fa2c940287bf1396bce195f541ebce2ad2b48